### ISSUE TITLE
ValidateFunc is for validating user input, there's nothing to validate on computed-only field

### ISSUE THREAD

**Issue by lizdeika** [GtheSheep/terraform-provider-dbt-cloud#45]
Hi,

I get these errors on plan. Not even using those data sources

```
Initializing Terraform configuration...
╷
│ Error: InternalValidate
│ 
│   with provider["registry.terraform.io/gthesheep/dbt-cloud"],
│   on main.tf line 39, in provider "dbt":
│   39: provider "dbt" {
│ 
│ Internal validation of the provider failed! This is always a bug
│ with the provider itself, and not a user issue. Please report
│ this bug:
│ 
│ 2 errors occurred:
│ 	* data source dbt_cloud_environment: type: ValidateFunc is for validating
│ user input, there's nothing to validate on computed-only field
│ 	* data source dbt_cloud_snowflake_credential: auth_type: ValidateFunc is
│ for validating user input, there's nothing to validate on computed-only
│ field
```

**Comment by GtheSheep** [comment]
hey, yeah I noticed this too and have removed here: https://github.com/GtheSheep/terraform-provider-dbt-cloud/pull/46/files#diff-e482239cbfbe720bb2063a13b26061f5df45b1d437f1390879a09129c647dcdeL32

will try and merge/ release today or tomorrow

**Comment by GtheSheep** [comment]
hey @lizdeika - I've merged that PR early in the hope it's fixed this issue, bumping to 0.0.71 locally and running init/ plan seems to have fixed it for me, let me know if this works for you 😸

**Comment by lizdeika** [comment]
It is working now, thanks!
